mainnet-1/specs/cosmoshub.json

```diff
@@ -0,0 +1,159 @@
+{
+  "proposal": {
+    "title": "Add Specs: Cosmos hub",
+    "description": "Adding new specification support for relaying cosmoshub data on Lava",
+    "specs": [
+      {
+        "index": "COSMOSHUB",
+        "name": "cosmos hub mainnet",
+        "enabled": true,
+        "imports": [
+          "COSMOSSDKFULL",
+          "COSMOSSDK45DEP"
```

Comment:
this is outdated cosmos hub is already version 50

Reply:
In cosmohub mainnet cosmossdk is v0.47 
In cosmosub testnet cosmossdk is v0.50  

We have only `COSMOSSDK` spec on the mainnet should we use only this or we need  something to add/change?